charts/astronomer/values.yaml

```diff
@@ -56,6 +56,7 @@ astroUI:
   #   memory: 128Mi
 
 houston:
+  prismaConnectionLimit: 5
```

Comment:
this seems like we are going with the mind set "everyone, including our corporate customers, are going to have a connection limit set to 5".

is that what we want? or do we want to add the ability to control it if they so choose and leave behavior the same if they don't?

Reply:
@tommyk we could set to `~` and you all can set it in cloud?  Does that work for you?

Reply:
my vote would be to not change the behavior and just allow users to override it if they choose.  

right now i think we are the only people really worried about this in our cloud, so changing it for everyone has a bigger potential ripple effect than leaving it the same imo. 

you guys can decide, i don't want to hold this up.